engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeServerTest.scala

```diff
@@ -946,9 +956,11 @@ class RuntimeServerTest
       context.Main.Update.mainZ(contextId),
       idMainUpdate,
       Api.Response(
-        Api.SuggestionsDatabaseReIndexNotification(
-          moduleName,
+        Api.SuggestionsDatabaseModuleUpdateNotification(
```

Comment:
Why does a suggestion DB carry the whole file in it? This seems very wasteful

Reply:
I also planned to address this in #1162, as the fix will require a bit of refactoring :+1: